scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestControllerTest.java

```diff
@@ -331,6 +330,41 @@ public void testCreateSchedule_multiple_error() throws Exception {
 		testCreateSchedule_defaultInstanceMinCount_greater_than_defaultInstanceMaxCount();
 	}
 
+	@Test
+	@Transactional
+	public void testDeleteSchedules() throws Exception {
+
+		// Test multiple applications each having multiple specific date schedules, no recurring schedule
+		String[] multipleAppIds = TestDataSetupHelper.generateAppIds(5);
+		assertDeleteSchedules(multipleAppIds, 5, 0);
+
+		// Note: Once recurring schedule delete in place following can be used 
```

Comment:
Should really be 3 tests rather than 3 in 1 but we seem to be paying a heavy cost in sprint startup for each test so it is fine.